en/docs/develop-components/configure-endpoints-body.md

```diff
@@ -0,0 +1,124 @@
+| **version**          | Required     | The version of the endpoints.yaml file.                                           |
+| **name**             | Required     | A unique name for the endpoint, which Choreo will use to generate the managed API.|
+| **port**             | Required     | The numeric port value that gets exposed via this endpoint.                      |
+| **type**             | Required     | The type of traffic this endpoint is accepting, such as `REST`, `GraphQL`, or `gRPC`.Currently the MI preset supports only the `REST` type.                                         |
+| **networkVisibility**| Required     | The network level visibility of this endpoint, which defaults to `Project` if not specified. Accepted values are `Project`, `Organization`, or `Public`.|
+| **context**          | Required     | The context (base path) of the API that Choreo exposes via this endpoint.        |
+| **schemaFilePath**   | Required     | The swagger definition file path. Defaults to the wildcard route if not provided. this field should be a relative path to the project path when using the MI build preset. For REST endpoint types when using the Ballerina or Dockerfile preset, this field should be a relative path to the component root or Docker context .|
+
+#### Sample endpoints.yaml
+
+**File location**:
+
+```bash
+<docker-build-context-path>/.choreo/endpoints.yaml
+```
+
+!!! note
+    - For components built with Ballerina build preset `docker-build-context-path` should be replaced with `component-root`. 
+    For example: `<component-root>/.choreo/endpoints.yaml`
+
+    - For components built with WSO2 MI build preset `docker-build-context-path` should be replaced with `<Project Path>`. 
+    For example: `<Project Path>/.choreo/endpoints.yaml`
+
+**File Content**:
+
+```yaml
+# +required Version of the endpoint configuration YAML
+version: 0.1
+
+# +required List of endpoints to create
+endpoints:
+  # +required Unique name for the endpoint. (This name will be used when generating the managed API)
+- name: Greeting 9090
+  # +required Numeric port value that gets exposed via this endpoint
+  port: 9090
+  # +required Type of the traffic this endpoint is accepting. Example: REST, GraphQL, etc.
+  # Allowed values: REST, GraphQL, GRPC
+  type: REST
+  # +optional Network level visibility of this endpoint. Defaults to Project
+  # Accepted values: Project|Organization|Public.
+  networkVisibility: Project
+  # +optional Context (base path) of the API that is exposed via this endpoint.
+  # This is mandatory if the endpoint type is set to REST or GraphQL.
+  context: /greeting
+  # +optional Path to the schema definition file. Defaults to wild card route if not provided
+  # This is only applicable to REST endpoint types.
+  # The path should be relative to the docker context.
+  schemaFilePath: greeting_openapi.yaml
+```
+
+## Expose endpoint as managed APIs
```

Comment:
Should be either 
Expose endpoints as managed APIs
or
Expose an endpoint as a managed API

Reply:
fixed